modules/basegl/glsl/background.frag

```diff
@@ -74,6 +74,12 @@ vec4 linearGradientVertical(vec2 texCoord) {
     return mix(bgColor2, bgColor1, texCoord.y);
 }
 
+vec4 linearGradientSpherical(vec2 texCoord) {
+	// bgColor1: inner color in circle
+	// bgColor2: outer color
+    return mix(bgColor1, bgColor2, distance(vec2(0.5), texCoord));
```

Comment:
This distance will at most be sqrt(.5) (distance from corner pixel to center pixel). It will never be 1, hence, ´bgColor2´ will never be fully represented. Should we handle that?